### ISSUE TITLE
WiFi not connecting, second attempt sends password twice

### ISSUE THREAD

**Issue by mozzwald** [FujiNetWIFI/fujinet-config#11]
When connecting to wifi for the first time I enter the access point password and it is sent to FujiNet but does not connect. It appears to also send a blank AP and Password after the initial correct AP/Password. When it fails to connect and you attempt again, it sends the password twice to FujiNet. Also worth noting that on the Atari the field where you type the password moves to the right on each attempt.

```
[08:18:02]Connecting to net: thegrid password: PASSWORD
[08:18:02]WiFi connect attempt to SSID "thegrid"
[08:18:02]esp_wifi_connect returned 0
[08:18:02]COMPLETE!
[08:18:02]
[08:18:02]CF: 70 fe 00 00 6f
[08:18:02]sioFuji::sio_process() called
[08:18:02]ACK!
[08:18:02]Fuji cmd: GET SSID
[08:18:02]->SIO write 96 bytes
[08:18:02]COMPLETE!
[08:18:02]
[08:18:02]CF: 70 fb 00 00 6c
[08:18:02]sioFuji::sio_process() called
[08:18:02]ACK!
[08:18:02]Fuji cmd: SET SSID
[08:18:02]<-SIO read 96 bytes
[08:18:02]ACK!
[08:18:02]Connecting to net:  password: 
[08:18:02]WiFi connect attempt to SSID ""
[08:18:02]esp_wifi_connect returned 12298
[08:18:02]COMPLETE!
[08:18:02]
[08:18:02]CF: 70 fa 00 00 6b
[08:18:02]sioFuji::sio_process() called
[08:18:02]ACK!
[08:18:02]Fuji cmd: GET WIFI STATUS
[08:18:02]->SIO write 1 bytes
[08:18:02]COMPLETE!
<SNIP>
[08:18:31]Connecting to net: thegrid password: PASSWORDPASSWORD
[08:18:31]WiFi connect attempt to SSID "thegrid"
[08:18:31]esp_wifi_connect returned 0
[08:18:31]COMPLETE!
```

**Comment by TheRobotFactory** [comment]
Myfirst thought is... is your wifi password really PASSWORDPASSWORD or is it passwordpassword? I got caught by the case sensitivity the first few times.

**Comment by mozzwald** [comment]
> Myfirst thought is... is your wifi password really PASSWORDPASSWORD or is it passwordpassword? I got caught by the case sensitivity the first few times.

My password is more like pAssword1 and I did verify case in the debug output. Fujinet got the correct password.

Would also be nice to have caps lock turned off by default in config also

**Comment by mozzwald** [comment]
Fixed with 150bcbf301ea13e9cfd1c720c1a6a7c6fb396b28
